controllers/articles.controllers.js

```diff
@@ -0,0 +1,17 @@
+const { selectArticle } = require("../models/articles.models");
+
+exports.getArticleById = ( req, res, next ) => {
+    const { article_id } = req.params;
+    selectArticle(article_id).then( ({ rows }) => {
+        const article = rows[0];
+        if(!article){
+            return Promise.reject({status: 404, msg: 'There is not an article at this ID sorry!'});
```

Comment:
Your promise.reject would usually be the realm of the of the model as it's job is the requesting of the data and if no data is coming back, it will be responsible for sending that error on to the catch block in the controller. 

Please amend this and test it and I am happy for you to merge :)